src/modules/pop_up_display.js

```diff
@@ -0,0 +1,94 @@
+import getComments from './pop_up_comments_get.js'; // will return an array with objets
+import postComments from './pop_up_comments_add.js';
+import commentsCounter from './pop_up_comments_counter.js';
+
+const setCardData = (countryName, imgUrl = '#', id, area, subRegion, population) => {
+  const popUpCard = `
+<div class="card">
+  <button type="button" class="close-pop-up">X</button>
+  <img src="${imgUrl}" alt="description here">
+  <h3>${countryName}</h3>
+  <div class="characteristics">
+      <span class="element">${id}</span>
+      <span class="element">${area}</span>
+      <span class="element">${subRegion}</span>
+      <span class="element">${population}</span>
+  </div>
+
+  <div class="comments-box">
+    <h3>Comments<span id="comments-counter"></span></h3>
+    <ul class="comments-list">
+      <li>
+          <span class="date">03/11/2021</span>
+          <span class="user">Alex Marin :</span>
+          <span class="comment">I'd love to buty it!</span>
+      </li>
+    </ul>
+  </div>
+
+  <form class="add-comments">
+    <input type="text" placeholder="Your name" id="your-name" >
+    <textarea placeholder="Your Insights" id="input-your-comments"></textarea>
+    <button type="button" class="submit-comment-btn">Comment</button type="button">
+  </form>
```

Comment:
- Great Job for implementing add a comment section, I think it would be better if update the screen to the newly added comment instead of closing the modal for better user interactivity as mentioned in the [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md)👍
![Screenshot 2023-06-30 073252](https://github.com/Diegogagan2587/JavaScript-Capstone-Project/assets/52010789/2315c883-82ec-47c3-ba26-d189ad7a6d64)

Reply:
@binhussen Thank you so much for the observation, We will apply the recommended suggestion! We do appreciate you pointing this out!